### ISSUE TITLE
Deviate lurkers are coming out of the water 

### ISSUE THREAD

**Issue by gingerbread2465** [elysium-project/server#1822]
![wowscrnshot_072917_210737](https://user-images.githubusercontent.com/11489723/28749825-808881f2-74a2-11e7-82e0-83dc53d18a6a.png)

**Comment by ifyousayso** [comment]
Yeah.. but all swimmers bounce out of it every now and then.. fishies in both Loch Modan and ZG do it.
